admin/commands/management/remove.go

```diff
@@ -45,6 +49,45 @@ type removeMySQLCommand struct {
 }
 
 func (cmd *removeMySQLCommand) Run() (commands.Result, error) {
+	if cmd.ServiceID == "" && cmd.ServiceName == "" {
+		// Automatic service lookup during removal
+		//
+		// Get services and remove it automatically once it's only one
+		// service registered
+		status, err := agentlocal.GetStatus(agentlocal.DoNotRequestNetworkInfo)
+		if err != nil {
+			return nil, err
+		}
+
+		servicesRes, err := inventoryClient.Default.Services.ListServices(&services.ListServicesParams{
+			Body: services.ListServicesBody{
+				NodeID:      status.NodeID,
+				ServiceType: cmd.serviceType(),
+			},
+			Context: commands.Ctx,
+		})
+		if err != nil {
+			return nil, err
+		}
+		switch {
+		case len(servicesRes.Payload.Mysql) == 1:
+			cmd.ServiceID = servicesRes.Payload.Mysql[0].ServiceID
+		case len(servicesRes.Payload.Mongodb) == 1:
+			cmd.ServiceID = servicesRes.Payload.Mongodb[0].ServiceID
+		case len(servicesRes.Payload.Postgresql) == 1:
+			cmd.ServiceID = servicesRes.Payload.Postgresql[0].ServiceID
+		case len(servicesRes.Payload.Proxysql) == 1:
+			cmd.ServiceID = servicesRes.Payload.Proxysql[0].ServiceID
+		case len(servicesRes.Payload.Haproxy) == 1:
+			cmd.ServiceID = servicesRes.Payload.Haproxy[0].ServiceID
+		case len(servicesRes.Payload.External) == 1:
+			cmd.ServiceID = servicesRes.Payload.External[0].ServiceID
+		}
+		if cmd.ServiceID == "" && cmd.ServiceName == "" {
+			return nil, errors.New("We can't find a service associated with the local node. service_id or service_name is required.")
```

Comment:
I suggest we use "Service ID" and "Service name" instead. The arguments may change in the future and we will forget to update this text.

```suggestion
			return nil, errors.New(`we could not find a service associated with the local node. Please provide "Service ID" or "Service name"`)
```